### PR TITLE
Feature/set back and front end

### PR DESCRIPTION
**Pull request by Salvador-ON**
- It has one model: User
- The User has 3 attributes: username, email, and password with the respective validation
- In the front-end, we can create new users or edit previous users

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+</p>
+<p>
+  <%= f.label :password, :value => "your password here:" %> <br/>
+  <%= f.password_field :password %>
+</p>
+<p><%= f.submit %></p>
+<% end %>
+
+<!--
+<%= form_tag("/users", method: "post") do %>
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: Please, use **`users_path`** instead of `/users`

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  <%= label_tag(:password, "password:") %>
+  <%= password_field_tag(:password) %><br>
+
+  <%= submit_tag("Submit") %>
+<% end %>
+-->
+
+<!-- 
+<form accept-charset="UTF-8" action="/users" method="post">
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: Please, use **`users_path`** instead of `/users`

File: app/views/users/new.html.erb (hunk 1 above)
Comment (Salvador-ON) [reply app/views/users/new.html.erb]: ok i will take care thanks

File: app/views/users/new.html.erb (hunk 2 above)
Comment (Salvador-ON) [reply app/views/users/new.html.erb]: ok i will take care 

thanks

**Pull request merged**